openmetadata-ui/src/main/resources/ui/src/pages/LoginPage/SignInPage.tsx

```diff
@@ -268,13 +268,14 @@ const SignInPage = () => {
                     </div>
                   </div>
                 )}
-                <div className="mt-8" onClick={onClickForgotPassword}>
-                  <Typography.Link underline data-testid="forgot-password">
-                    {t('label.forgot-password')}
-                  </Typography.Link>
-                </div>
-
-                {(authConfig?.enableSelfSignup || isAuthProviderLDAP) && (
+                {(!isAuthProviderLDAP) && (
+                  <div className="mt-8" onClick={onClickForgotPassword}>
+                    <Typography.Link underline data-testid="forgot-password">
+                      {t('label.forgot-password')}
+                    </Typography.Link>
+                  </div>
+                )}
+                {(authConfig?.enableSelfSignup) && (!isAuthProviderLDAP) && (
```

Comment:
@chirag-madlani lets change based on one config. Why are we making it allowable for if its LDAP